core/modules/installer/installer.info

```diff
@@ -0,0 +1,7 @@
+name = Installer
+description = Securely installs or updates modules, themes, and layouts via a web interface.
+version = BACKDROP_VERSION
+type = module
+package = Development
+backdrop = 1.x
+dependencies[] = update
```

Comment:
Does it really depend on update module?  Can't you use the UI to install new things without checking for updates? (I will check the code, but it would be nice if this was not a dependency.)